app/src/main/java/com/example/parkingsystem/mvp/presenter/ConfigureParkingDialogPresenter.java

```diff
@@ -0,0 +1,32 @@
+package com.example.parkingsystem.mvp.presenter;
+
+import android.text.TextUtils;
+import com.example.parkingsystem.fragment.ConfigureParkingDialogListener;
+import com.example.parkingsystem.mvp.contracts.ConfigureParkingDialogContract;
+
+public class ConfigureParkingDialogPresenter implements ConfigureParkingDialogContract.ConfigureParkingDialogPresenter {
+    private final ConfigureParkingDialogContract.ConfigureParkingDialogView view;
+    private final ConfigureParkingDialogListener listener;
+
+    public ConfigureParkingDialogPresenter(ConfigureParkingDialogListener listener, ConfigureParkingDialogContract.ConfigureParkingDialogView view) {
+        this.view = view;
+        this.listener = listener;
+    }
+
+    @Override
+    public void onButtonDialogParkingConfirmPressed() {
+        String parkingSpaces = view.getParkingSpaces();
+        if (!TextUtils.isEmpty(parkingSpaces)) {
+            int intParkingSpaces = Integer.parseInt(parkingSpaces);
+            view.closeDialog();
+            listener.onDialogPositiveClick(intParkingSpaces);
```

Comment:
this call should be in the view, not here in the presenter

Reply:
Done!